tests/unit/submissions/test_authors.py

```diff
@@ -554,22 +554,34 @@ def test_load_author_project_membership():
     assert result == expected
 
 
-def test_dump_author_public_emails():
-    data = {**DEFAULT_DATA_TO_DUMP, "email_addresses": [{"value": "email@email.com"}]}
+def test_dump_author_emails():
+    data = {
+        **DEFAULT_DATA_TO_DUMP,
+        "email_addresses": [{"value": "email@email.com", "hidden": True}],
+    }
 
     record = faker.record("aut", data=data)
 
     result = Author().dump(record).data
-    expected = {**DEFAULT_DATA_DUMP, "public_emails": ["email@email.com"]}
+    expected = {
+        **DEFAULT_DATA_DUMP,
+        "emails": [{"value": "email@email.com", "hidden": True}],
+    }
 
     assert result == expected
 
 
 def test_load_author_public_emails():
-    data = {**DEFAULT_DATA_TO_DUMP, "public_emails": ["email@email.com"]}
+    data = {
+        **DEFAULT_DATA_TO_DUMP,
+        "emails": [{"value": "email@email.com", "hidden": True}],
+    }
 
     result = Author().load(data).data
-    expected = {**DEFAULT_DATA_LOAD, "email_addresses": [{"value": "email@email.com"}]}
+    expected = {
+        **DEFAULT_DATA_LOAD,
+        "email_addresses": [{"value": "email@email.com", "hidden": True}],
+    }
```

Comment:
don't we need to add a case without ``hidden`` field? 🤔

Reply:
we're not doing anything special for it, but sure.